### DISCUSSION TITLE
NewRelic with inboard

### DISCUSSION THREAD

**Discussion by flpStrri** [br3ndonland/inboard#101] [Q&A] [answered]
What would be the best way to inject the NewRelic admin to the Docker entry point? I thought about changing the `ENTRYPOINT` to `newrelic-admin run-program python` but I am not sure if it is the best idea.

**Comment by br3ndonland** [comment, marked as answer]
The inboard Dockerfile uses both `ENTRYPOINT` and `CMD`. When `ENTRYPOINT` and `CMD` are used together, typically executable programs are specified in `ENTRYPOINT` and program options are specified in `CMD`. [If you override the `ENTRYPOINT` from a base image (like inboard), you also need to override the `CMD`](https://docs.docker.com/reference/dockerfile/#understand-how-cmd-and-entrypoint-interact).

Based on the [NewRelic docs](https://docs.newrelic.com/install/python/?python-non-web=non-web-no&python-docker=docker-yes&python-framework=fastapi&python-server=uvicorn), it looks like you could override the `ENTRYPOINT` within your Dockerfile like this:

```dockerfile
ENTRYPOINT ["newrelic-admin", "run-program", "python"]
CMD ["-m", "inboard.start"]
```
